### ISSUE TITLE
Make the Battery State in % not V

### ISSUE THREAD

**Issue by janhp** [ConnectorGit/ioBroker.connector-local-control#1]
Dear Dev,

it would be awesome if the battery state/batteryLevel would be in % and not in V.

Appreciate the update!

Thanks a lot.

**Comment by deichstaedt** [comment]
Good idea! An additional battery low indicator would also be helpful.

**Comment by somansch** [comment]
Agree, make a DP called "battery" in % (like in other iobroker adapters)

**Comment by Eelisglitchy** [comment]
*Push, plz put a update.
